server/models/index.js

```diff
@@ -17,7 +17,8 @@ sequelize = new Sequelize(
             ssl: true,
             native: true
         },
-        logging: false // console.log // eslint-disable-line
+        // logging: console.log // eslint-disable-line
+        logging: false
```

Comment:
Guessing you want to revert these 2 lines

Reply:
Good guess, but actually this is intentional! I noticed that it's hard to quickly turn logging on and off. I figured out that it would be faster and easier to have two lines where you can comment one and uncomment the other. So that's why I changed this.

Reply:
Got it. Well wouldn't it be better to control logging behind a flag then? Seems very easy as something to have future merge conflicts if someone forgets to revert.

Since something like this only has value in a dev environment. I figure doing `yarn dev-debug` could enable that flag vs `yarn dev`.

Reply:
I'll just revert it